Configure.pri.in

```diff
@@ -17,9 +17,6 @@ CONFIG += @HAS_LOG4CXX@ @HAS_NEWMAT@ @HAS_CPPUNIT@ \
 CONFIG += release
 CONFIG -= debug
 
-# We don't use multiple threads in Hootenanny, speed things up a bit.
-DEFINES += BOOST_SP_DISABLE_THREADS
-
 exists(LocalConfig.pri):include(LocalConfig.pri)
```

Comment:
This is a crucial change that was needed, and I originally wasn't aware of it.  Without this, the synchronization of the smart pointers is unpredictable.